app/controllers/reports/regions_controller.rb

```diff
@@ -110,6 +111,27 @@ def download
     end
   end
 
+  def whatsapp_graphics
```

Comment:
Do you want to stick a quick controller spec for this action?

Reply:
Yup done. 👍